lms/djangoapps/teams/static/teams/templates/team-membership-details.underscore

```diff
@@ -1,4 +1,4 @@
-<span class="member-count"><%= membership_message %></span>
+<span class="member-count"><%= HtmlUtils.ensureHtml(membership_message) %></span>
```

Comment:
Another place where I wonder if it would make more sense to just have the message in here?